scylla-cql/src/frame/response/result.rs

```diff
@@ -49,9 +49,50 @@ pub struct TableSpec<'a> {
     table_name: Cow<'a, str>,
 }
 
+/// A type of the column in schema metadata or in prepared statement.
+///
+/// In schema metadata this represents simply a column type. In prepared
+/// statement this is used in two places: as a type of bind marker and as a type
+/// of returned value.
+///
+/// Some of the variants contain a `frozen` flag. This flag is only used
+/// in schema metadata. For prepared statement bind markers and result types
+/// those fields will always be set to `false` (even if the DB column
+/// corresponding to given marker / result type is frozen).
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub enum ColumnType<'frame> {
```

Comment:
:thinking: I believe the `ColumnType` is used in the result set of both prepared and unprepared statements. The docstring specifically mentions only prepared statements, which I'm afraid is misleading.